components/uid-input/src/UidInput.ts

```diff
@@ -77,6 +76,10 @@ export class UidInput extends LabeledElement {
       `
     ];
 
+  async attributeChangedCallback(name: string, old: string | null, value: string | null): Promise<void> {
```

Comment:
Is this required to implement this function? as it only call the Super?

Reply:
No, this is indeed not needed